preprocessors/collage-detector.schema.json

```
@@ -0,0 +1,11 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/collage-detector.schema.json",
+    "type": "object",
+    "title": "Collage Detector",
+    "description": "Binary Classifier for collages. Accessed as 'ca.mcgill.a11y.image.preprocessor.collageDetector'.",
+    "properties": {
+        "category": { "enum": ["collage", "notCollage"] }
```

Considering this is a binary classifier and, at least to me, has no path to actually having multiple possible output values, this should probably be changed to a boolean.
```suggestion
        "collage": {
            "type": "boolean",
            "description": "If the input graphic was classified as a collage or not."
        }
```